docs/book/src/quinn.md

```diff
@@ -0,0 +1 @@
+{{#include ../../../README.md}}
```

Comment:
Can you do this as a rename?

Reply:
What do you mean by that?

Reply:
This PR removes and re-adds `quinn.md`. It would be better if Git understood it as a rename.

Reply:
mmm. I moved it to a subfolder. First, all contents were in `./docs` now in `./docs/book`. It is strange it did not account for this move as a move. Let me check interwebs if there is a workaround to trick git

Reply:
Ah I see https://stackoverflow.com/questions/433111/how-to-make-git-mark-a-deleted-and-a-new-file-as-a-file-move. 

The reason it does this is that I move and 'change' the file. It needs this change because the README is now one folder higher then before.

Reply:
I think we should go with it for now. Better now than later.